src/status_im2/contexts/communities/menus/community_options/view.cljs

```diff
@@ -151,14 +152,14 @@
   [{:keys [id]}]
   (let [community     (rf/sub [:communities/community id])
         token-gated?  (:token-gated? community)
-        joined?       (:joined community)
         admin?        (:admin community)
+        joined?       (:joined community)
```

Comment:
To make renaming more concise, it can be done along with destructuring:
`(let [{admin? :admin joined? :joined banned? :banList} community])`